### PR TITLE
Fix crash when receiving a protomsg with an unexpected type.

### PR DESCRIPTION
**Pull request by kacf**
Changelog: None

Signed-off-by: Kristian Amlie <kristian.amlie@northern.tech>

### CODE REVIEW

File: app/daemon.go [hunk 1]
```diff
@@ -483,7 +483,9 @@ func (d *MenderShellDaemon) readMessage(webSock *connection.Connection) (*shell.
 
 	status := wsshell.NormalMessage
 	if v, ok := msg.Header.Properties["status"]; ok {
-		status = wsshell.MenderShellMessageStatus(v.(int64))
+		if vint64, ok := v.(int64); ok {
+			status = wsshell.MenderShellMessageStatus(vint64)
+		}
```
Comment (merlin-northern) [inline app/daemon.go:488]: can we get an else with some `Debugf`?

**Review by merlin-northern — APPROVED** [review]
lgtm one small request

**Comment by kacf** [comment]
Good idea, added!

**Review by merlin-northern — APPROVED** [review]
lgtm!

**Comment by kacf** [comment]
Merging. The master branch failure is the usual `fw_setenv` failure that we already have a ticket for, and the staging pipelines are irrelevant to this PR.

**Pull request merged**